app/models/condition.rb

```diff
@@ -32,6 +32,16 @@ def truncated_comment
     (text + "\n\nReason: #{reason}").truncate(100, separator: "Reason")
   end
 
+  class << self
+    def standard_conditions
+      I18n.t(:conditions_list).map do |k, v|
+        c = Condition.new(v)
+        c.standard = true
+        c
+      end
+    end
```

Comment:
Isn't `standard` set to `true` in the locale file?

Reply:
Oh yes, seems unnecessary. Will remove.